script/deploy-canary-demo

```diff
@@ -2,10 +2,12 @@
 set -euxo pipefail
 
 cd $GITHUB_WORKSPACE/demos/browser
+node script/conditional-install
```

Comment:
Doesn't npm run build also run conditional-install?

Reply:
Ah, that's why I didn't have it in there lol